app/views/candidate_interface/carry_over/start_between_cycles_unsubmitted.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <% if CycleTimetable.before_apply_reopens? %>
```

Comment:
See above — this is inconsistent because it's possible to be `before_apply_reopens?` before we've entered the new cycle

Reply:
These carry over interstitials are only displayed if Time.now is `between_cycles_apply_2` - see https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/controllers/candidate_interface/carry_over_controller.rb#L7

I think perhaps some more tweaks / refactoring may be required when we come to updating the interstitials (which are separate cards)